src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/memories/Comment.kt

```diff
@@ -0,0 +1,6 @@
+package com.atlassian.performance.tools.jiraactions.api.memories
+
+data class Comment(
+    val id: String,
```

Comment:
This is probably insignificant, but should we actually remember comment ID? Are users doing that? When I'm relating to an issue comment I only use URL. The ID seems to be a Jira's implementation details.

Nevertheless we are remembering ID's already in memories for other data types, so maybe it's fine.

Reply:
I didn't think about comment memory in that way. Sure, users doesn't care about comment id. I've added it here because it was convenient to have this whole parsing html in one place (while creating Comment object). I'll consider moving comment id extraction to a different place, if it's not too complicated.

Reply:
We discussed the matter via zoom call. We agreed that `id` can stay. This is mainly, so that we can proceed with the merge ASAP.

Jira's implementation details around this is highly unlikely to change and we are using it already when validating that the comment was focused.